ipatests/util.py

```diff
@@ -328,20 +328,20 @@ def assert_deepequal(expected, got, doc='', stack=tuple()):
     If the tests fails, it will raise an ``AssertionError`` with detailed
     information, including the path to the offending value.  For example:
 
-    >>> expected = [u'Hello', dict(world=u'how are you?')]
-    >>> got = [u'Hello', dict(world='how are you?')]
+    >>> expected = [u'Hello', dict(world=1)]
+    >>> got = [u'Hello', dict(world=1.0)]
```

Comment:
The old test scenario does no longer work on Python 3 since ```u'how are you'``` and ```'how are you'``` have identical type. ```u'how are you' != b'how are you'```. It works with int / float on all Python versions.